src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -133,9 +133,10 @@ var inputValueTracking = {
       return false;
     }
     var tracker = getTracker(subject);
+    const isNode = (subject: any).nodeType === ELEMENT_NODE;
 
     if (!tracker) {
-      if (typeof (subject: any).tag === 'number') {
+      if (isNode || typeof (subject: any).tag === 'number') {
```

Comment:
@jquense but in this case you're always passing `subject.stateNode` to `trackNode`. If `subject` is a DOM node (which the `isNode` check allows) `subject.stateNode` will be `undefined`.

I think we should do `inputValueTracking.trackNode((subject: any))` if `isNode` is true, right?